### PR TITLE
Krknlib prometheus client integration + kube_burner removed

### PR DESCRIPTION
**Pull request by tsebastiani**

### CODE REVIEW

**Comment by paigerube14** [comment]
Tullio, I ran this code and the output looks multiplied 

```
2023-12-05 11:12:01,408 [INFO] Alerts checking is enabled
2023-12-05 11:12:01,649 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.007737154071484073s
2023-12-05 11:12:01,649 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.007737154071484073s
2023-12-05 11:12:01,649 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.007737154071484073s
2023-12-05 11:12:01,649 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.007737154071484073s
2023-12-05 11:12:01,649 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.007737154071484073s
2023-12-05 11:12:01,649 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.007737154071484073s
2023-12-05 11:12:01,649 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.007737154071484073s
2023-12-05 11:12:01,649 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.007737154071484073s
2023-12-05 11:12:01,650 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.007737154071484073s
2023-12-05 11:12:01,650 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.007737154071484073s
2023-12-05 11:12:01,650 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.007989670544270548s
2023-12-05 11:12:01,650 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,650 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,650 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,650 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,650 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,650 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,651 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,651 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,651 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,651 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,651 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,651 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,651 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,651 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,651 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,651 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,651 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,652 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,652 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,652 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,652 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,652 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,652 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,652 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,652 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,652 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,652 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,653 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,653 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008028754071484075s
2023-12-05 11:12:01,653 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.00831500387760388s
2023-12-05 11:12:01,653 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,653 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,653 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,653 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,653 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,653 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,653 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,653 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,653 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,653 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,654 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,654 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,654 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,654 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,654 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,654 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,654 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,654 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,654 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,654 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,654 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,654 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,666 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,666 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,666 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,666 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,667 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,667 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,669 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,669 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008366935889665894s
2023-12-05 11:12:01,669 [WARNING] 10 minutes avg. 99th etcd commit latency on etcd-ip-10-0-49-160.us-east-2.compute.internal higher than 30ms. 0.008418475025559239s
2023-12-05 11:12:02,714 [INFO] Successfully finished running Kraken. UUID for the run: bf1109d6-5ddd-497d-b4b3-59dc601402a8. Report generated at /Users/prubenda/PycharmProjects/pplRepos/tsebastiani/krkn/kraken.report. Exiting
```

File: scenarios/arcaflow/cpu-hog/config.yaml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ---
 deployer:
-  connection: {}
+  connection:
+    cacert: ''
+    cert: ''
+    host: https://api.tsebasti-lab.aws.rhperfscale.org:6443
```
Comment (chaitanyaenr) [inline scenarios/arcaflow/cpu-hog/config.yaml:6]: s/tsebasti-lab/example-cluster

File: run_kraken.py [hunk 2]
```diff
@@ -374,10 +380,10 @@ def main(cfg):
         # if platform is openshift will be collected
         # Cloud platform and network plugins metadata
         # through OCP specific APIs
-        if config["kraken"]["distribution"] == "openshift":
-            telemetry_ocp.collect_cluster_metadata(chaos_telemetry)
-        else:
-            telemetry_k8s.collect_cluster_metadata(chaos_telemetry)
+        # if config["kraken"]["distribution"] == "openshift":
```
Comment (chaitanyaenr) [inline run_kraken.py:383]: Did we disable it for testing purposes? Thoughts?

File: config/alerts.yaml [hunk 3]
```diff
@@ -0,0 +1,90 @@
+# etcd
```
Comment (chaitanyaenr) [inline config/alerts.yaml:1]: We need to update the docs here: https://github.com/redhat-chaos/krkn/blob/main/docs/metrics.md#scraping-and-storing-metrics-for-the-run.

File: config/config.yaml [hunk 4]
```diff
@@ -59,7 +59,7 @@ performance_monitoring:
     prometheus_bearer_token:                              # The bearer token is automatically obtained in case of OpenShift, please set it when the distribution is Kubernetes. This is needed to authenticate with prometheus.
     uuid:                                                 # uuid for the run is generated by default if not set
     enable_alerts: False                                  # Runs the queries specified in the alert profile and displays the info or exits 1 when severity=error
-    alert_profile: config/alerts                          # Path or URL to alert profile with the prometheus queries
```
Comment (chaitanyaenr) [inline config/config.yaml:62]: Need to remove the kube-burner reference: https://github.com/redhat-chaos/krkn/blob/main/config/config.yaml#L54

File: run_kraken.py (hunk 2 above)
Comment (tsebastiani) [reply run_kraken.py:383]: yeah, an oversight, restoring.

**Review by chaitanyaenr — APPROVED** [review]
LGTM!

**Pull request merged**